src/index.js

```diff
@@ -1,29 +1,158 @@
 import './style.css';
 
-const listArray = [{
-  description: 'wash the dishes',
-  complete: true,
-  index: 0,
-},
-{
-  description: 'complet To Do list project',
-  complete: false,
-  index: 1,
-},
-];
+const textInput = document.querySelector('input');
+const listToDo = document.querySelector('.listToDo');
+const clearAllbtn = document.querySelector('button');
 
-const listToDo = document.getElementById('.myListToDo');
+class MyObject {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+const myArray = [];
+const updateLocal = () => {
+  const localData = JSON.parse(localStorage.getItem('list'));
+  const toDos = document.querySelectorAll('span');
+  for (let i = 0; i < toDos.length; i += 1) {
+    if (toDos[i].classList.contains('checkToDo')) {
+      localData[i].completed = true;
+    } else {
+      localData[i].completed = false;
+    }
+  }
+  localStorage.setItem('list', JSON.stringify(localData));
+};
+const removeToDo = (toDo) => {
+  listToDo.removeChild(toDo);
+  const localData = JSON.parse(localStorage.getItem('list'));
+  const data = Array.from(localData).filter((i) => i.completed === false);
+  data.forEach((el, index) => {
+    el.index = index + 1;
+  });
+  localStorage.setItem('list', JSON.stringify(data));
+};
```

Comment:
- [ ] For some reason I was not able to delete a todo that I had created. Kindly check as the "delete" icon does not show when I tap onto a todo.